examples/crm/src/activity/ActivityLogNote.tsx

```diff
@@ -1,8 +1,8 @@
-import Button from '@mui/material/Button';
 import ListItem from '@mui/material/ListItem';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
-import { ReactNode, useState } from 'react';
+import { MouseEventHandler, ReactNode, useState } from 'react';
+import { Link } from 'react-admin';
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
**note (non-blocking):** ‏It shouldn't be a Mui Link ?